src/ontology/cl-edit.owl

```diff
@@ -21950,14 +21951,14 @@ SubClassOf(obo:CL_0005011 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0046717))
 
 # Class: obo:CL_0005012 (multi-ciliated epithelial cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "CL:CVS") obo:IAO_0000115 obo:CL_0005012 "A ciliated epithelial cell with many cilia.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "CL:CVS") Annotation(oboInOwl:hasDbXref "PMID:28400610") Annotation(oboInOwl:hasDbXref "PMID:38032388") obo:IAO_0000115 obo:CL_0005012 "A columnar/cuboidal epithelial cell with multiple motile cilia on its apical surface. These cells facilitate the movement of liquids such as mucus or cerebrospinal fluid across the epithelial surface.")
 AnnotationAssertion(oboInOwl:hasBroadSynonym obo:CL_0005012 "multi-ciliated cell"@en)
 AnnotationAssertion(oboInOwl:hasBroadSynonym obo:CL_0005012 "multiciliated cell"@en)
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0005012 "multiciliated epithelial cell"@en)
 AnnotationAssertion(Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_0005012 "MCC"@en)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22118931") Annotation(oboInOwl:hasDbXref "PMID:23959957") Annotation(oboInOwl:hasDbXref "PMID:25729351") Annotation(oboInOwl:hasDbXref "PMID:26378583") rdfs:comment obo:CL_0005012 "While the term 'multi-ciliated epithelial cell'  refers to cells with multiple motile cilia, there are exceptions where epithelial cells may have multiple non-motile cilia. Examples include olfactory sensory neurons, which have non-motile cilia for sensory detection, and choroid plexus epithelial cells, which possess multiple non-motile cilia twith transient motility during development.")
```

Comment:
Typo last sentence, should be "cilia with transient"

Reply:
in the comment you might want to include how multi-ciliated cells are involved in health and disease
You might want to refer to this section `Multiple motile cilia-related ciliopathies` in the reference you used (PMID: 38032388[https://pubmed.ncbi.nlm.nih.gov/38032388/]) to briefly mention the ciliopathies  if you want

Reply:
I have added some information about ciliopathies in the comment section